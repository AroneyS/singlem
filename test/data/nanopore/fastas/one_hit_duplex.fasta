>one_hit_duplex
ATGAATATCCGGCAGTTTTTAACCGAGATCGGGAACCATATCCAGGATATTAACAGTAGCTGAAGTTACTGACTTACGTTCACAATTACGTGAAGCTGGTGTTGAGTATAAAGTATACAAAAACACTATGGTACGTCGTGCAGCTGAAATGAATATCCGGCAGTTTTTAACCGAGATCGGGAACCATATCCAGGAATTCCGAAAGGGGAAATCTTCCATGGACATGAATTCCACTACTCCCGACTGGAACTGGAGGGGGCCCAACCAAAATTTGCCTTTGATATCCTGCGTGGTAGGGGAGTCACTGGTAACTGTGATGGGCTCATGAGTAAAAACACATTGGCCAGTTACGTGCATACCCATGTGGCAGCTTGCCCCTCCTTTGCCAGTAGAATGGTGAAGGCGGCTGCCCATGATCTTTAATGAGGATTGAGAAGGAAAAAACTTTTTCAATACTTTAATTTTTAAAAAAGCCCTTTTGCTACCTTTTAAAATATTTTCTACTGTAGAATTAAATCAACCAATAATTGGATTGAATAACCTCTTATGGTTAATTAACTCTAATTAATATTTCTACTGTAGACTAAATTACCTTGCATTTTCACTTCATGGTAGTCTACTGTAGAATAAATTTGGTACAACTATCCTGGATATGGTTCCCGATCTCGGTTAAAAACTGCCGGATATTCATTTCAGCTGCACGACGTACCATAGTGTTTTTGTATACTTTATACTCAACACCAGCTTCACGTAATTGTGAACGTAAGTCAGTAACTTCAGCTACTGTTAATATCCTGGATATGGTTCCCGATCTCGGTTAAAAACTGCCGGATATTCAT
