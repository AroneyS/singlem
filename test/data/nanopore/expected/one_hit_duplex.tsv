gene	sample	sequence	num_hits	coverage	taxonomy
4.11.22seqs	one_hit_duplex	TTACGTTCACAATTACGTGAAGCTGGTGTTGAGTATAAAGTATACAAAAACACTATGGTA	1	1.08	
